modules/xmpp/ChatRoom.js

```diff
@@ -148,6 +148,16 @@ export default class ChatRoom extends Listenable {
         // Jicofo makes decisions based on that. Initialize it with 'false'
         // here.
         this.addVideoInfoToPresence(false);
+
+        if (options.deploymentInfo && options.deploymentInfo.userRegion) {
+            this.presMap.nodes.push(
+                { 'tagName': 'region',
```

Comment:
i don't know if we have a style for this, i feel like i typically see a newline after the opening brace of the object, but i don't know if we're consistent.  @virtuacoplenny ?